dbms/src/test/java/org/polypheny/db/misc/DataMigratorTest.java

```diff
@@ -71,7 +82,7 @@ public void basicTest() throws SQLException {
                             ImmutableList.of(
                                     new Object[]{ 1, 5, "foo" } ) );
                 } finally {
-                    // Drop table and store
+                    // Drop table and storeId
```

Comment:
Looks like a find and replace error

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520476075